pkg/cryptoauthlib/Makefile.cryptoauthlib

```diff
@@ -0,0 +1,3 @@
+MODULE = cryptoauthlib
+
+include $(RIOTBASE)/Makefile.bas
```

Comment:
I tried building without it and it works. I put it in there because all the other packages have it in their module makefiles. Maybe someone should check, if they're useless there as well ;-)

Reply:
Looking into `Makefile.base` (not its `base` not `bas` suffix) there are lots of macros and stuff that might be needed at some point (even though it works now). 

Also I would rather opt for consistency, i.e. matching other packages.

Further, `Makefile.base` is pulled in by many other modules (boards, cpus, sys/*) too.

Reply:
Thx, I'll keep it, then